packages/salesforcedx-vscode-soql/package.json

```diff
@@ -121,6 +121,20 @@
         "scopeName": "source.soql",
         "path": "./node_modules/@salesforce/apex-tmlanguage/grammars/soql.tmLanguage"
       }
-    ]
+    ],
+    "commands": [
+      {
+        "command": "soql.builder.open.new",
+        "title": "SOQL: Create Query in SOQL Builder"
```

Comment:
Why do we want to prefix this command with `SOQL: ` instead of using `SFDX:` like the other commands in the other extensions ?

Reply:
Move this label to a `package.nls.json` e.g. https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-core/package.nls.json

Reply:
I've moved this label and also the editor display name to `package.nls.json` in the `chore(soql): i18n in package.json and cleanup` commit.

Also, I've changed the command title to have an 'SFDX' prefix. The story on that was that at first I created a command title to be as brief as possible but I wanted to make sure it was searchable on both "SOQL and "Query" ("SOQL: New Query"). We went back and forth and landed on "Create Query in SOQL Builder" but never changed the prefix (until now!).